api/app/signals/apps/signals/managers.py

```diff
@@ -569,6 +569,18 @@ def update_multiple(self, data, signal):  # noqa: C901
                     'prev_type': previous_type
                 }))
 
+            if 'directing_department_assignment' in data:
+                previous_directing_departments = locked_signal.directing_departments_assignment
+                directing_departments = self._update_directive_departments_no_transaction(
```

Comment:
see below about naming

Reply:
Forgot to change this to match the rest of the feature, oops 😂